itou/siaes/models.py

```diff
@@ -486,6 +514,10 @@ def display_name(self):
     def get_absolute_url(self):
         return reverse("siaes_views:job_description_card", kwargs={"job_description_id": self.pk})
 
+    @cached_property
+    def is_popular(self):
+        return self.jobapplication_set.pending().count() > 20
```

Comment:
Je pense que tu n'as plus besoin de cette `property`, si ?

Reply:
Très juste ! Il s'agit d'un oubli. Merci de l'avoir noté, je l'enlève tout de suite.